packages/affiliates/README.md

```diff
@@ -0,0 +1,15 @@
+# Token Distributions
```

Comment:
nit: should you link these to particular directories? Either by specifying the names of the dirs inline or by adding a link?

Reply:
ye, there is a lot of stuff we could add to these readmes. We should think of a standard set of information that each package readme should contain and what the best way is to present the info.